README.md

```diff
@@ -1,14 +1,58 @@
 
-This wrapper repository creates a VPC to add these components in and a 'bastion host' for secure direct connections to the database (see the [asdi repository](https://github.com/developmentseed/aws-asdi-pgstac))
+## Automated Deployment
+
+Deployment happens through a github workflow manually triggered and defined in `.github/workflows/deploy.yaml`.
+
+## Networking and accessibility of the database. 
+
+Because of security requirements, the networking set up imposes the following constraints : 
+
+- For security reasons, the database is in a _private_ subnet of the VPC. As such, only instances running inside of the same VPC can access the database. This means that, for example, even if a user has the password and her IP is allowed inbound connections to the database, access will _not_ be allowed. 
+
+This has three consequences : 
+
+1. The APIs that need access to the database (the STAC API, the tiling API, the ingestion API) need to be deployed in that same VPC. 
+2. In addition, because these APIs _also_ sometimes need access to the internet, a NAT gateway must in addition be deployed in that VPC. 
+3. For direct, administrative connections to the database, one _must_ go through an instance placed in the same VPC as the database. 
+
+We approach (3) by re-using the 'bastion host' eoAPI construct, which deploys an EC2 instance that can connect to the database, and can be used to create a tunnel from a user's machine to the database. See the eoAPI docs for more information. 
+
+
+## Ingestion
+
+The term "ingestion" refers to the process of cataloging data in the STAC catalog associated with this deployment. 
+
+### Direct ingestion
+
+For a small record ingestion (for example a collection record or just one item), one can directly connect to the database and perform loading. This can be done using the `pypgstac` library. For example, to load an item stored locally in `test_item.json`, with `pypgstac` installed, you can run the following command : 
+
+```
+pypgstac load --table items test_item.json
+```
+
+or for a collection
+
+```
+pypgstac load --table collections test_collection.json
+```
+
+This requires 
+
+1. that you are allowed to connect to the database. Because of the security requirements mentioned above, you must go through a tunnel using the bastion host. [See these docs for more details](https://developmentseed.org/eoapi-cdk/#bastionhost-)
+2. [the configuration](https://stac-utils.github.io/pgstac/pypgstac/) for the database connection is present in your environment. 
```

Comment:
I think we want to avoid direct connections to the database and bastion host if we can help it

Reply:
Agreed, but shouldn't it be documented nevertheless?